cx-server-companion/life-cycle-scripts/cx-server

```diff
@@ -27,7 +34,7 @@ fi
         --env DEVELOPER_MODE \
         --env host_os=unix \
         --env cx_server_path="${PWD}" \
-        --env-file custom-environment.list \
+        ${custom_env_file_parameter} \
```

Comment:
Suggestion: Turn parameters into array and optionally append the env_file parameter. Would be cleaner from my perspective.

Reply:
Not sure if this was what you had in mind @benhei

Reply:
Yes, exactly that :)